battleship.py

```diff
@@ -114,7 +114,15 @@ def checkShip(grid, ship):
 Returns: 2D list of ints
 '''
 def addShips(grid, numShips):
-    return
+    board=grid

```

Comment:
This assignment is irrelevant and incorrect. Please work on the grid variable directly.